src/mlpack/methods/linear_regression/linear_regression_main.cpp

```diff
@@ -139,31 +142,26 @@ static void mlpackMain()
 
       if (responses.n_cols != regressors.n_cols)
       {
-        Log::Fatal << "The responses must have the same number of rows as the "
-            "training set." << endl;
+        Log::Fatal << "The responses must have the same number of columns "
+            "as the training set." << endl;
       }
     }
 
     Timer::Start("regression");
-    lr = LinearRegression(regressors, responses);
+    lr = LinearRegression(regressors, responses, lambda);
```

Comment:
Yes, this is another nice catch, it looks like `lambda` would have done nothing before.